docker-compose.yml

```diff
@@ -86,7 +87,7 @@ services:
   # Access the cli by running:
   # > docker-compose exec ksqldb-cli  ksql http://primary-ksqldb-server:8088
   ksqldb-cli:
-    image: ${KSQL_IMAGE_BASE}confluentinc/ksqldb-cli:${KSQL_VERSION}
+    image: confluentinc/ksqldb-cli:0.23.1
```

Comment:
i dont think we should be hard coding this, was the $KSQL_VERSION substitution not working for you?

Reply:
You're correct! I made a different mistake (`docker exec` instead of `docker-compose exec`) that resulted in not having a ptty. I'll update this now!